tests/test_avgHR.py

```diff
@@ -25,6 +25,15 @@ def test_avghr():
     times3 = list3[0]
     voltages3 = list3[1]
 
+    HRM1 = HRM(times1,voltages1)
+    HRM2 = HRM(times2, voltages2)
+    HRM3 = HRM(times3, voltages3)
+
+    HRM1.avghr()
+    HRM2.avghr()
+    HRM3.avghr()
+
+
```

Comment:
[nit] instead of cutting-n-pasting identical lines of code for each of the test cases, I would recommend putting the testing entities in a loop, and then loop over passing in different CSV files.  This will help if you change the progression of method calls in the future, and lets you also more easily expand to a larger number of test input files in the future.